src/detect-pcre.c

```diff
@@ -74,36 +74,19 @@ static int pcre_match_limit_recursion = 0;
 static DetectParseRegex parse_regex;
 static DetectParseRegex parse_capture_regex;
 
-#ifdef PCRE_HAVE_JIT
-static int pcre_use_jit = 1;
+#ifdef PCRE2_HAVE_JIT
+static int pcre2_use_jit = 1;
 #endif
 
-#ifdef PCRE_HAVE_JIT_EXEC
-#define PCRE_JIT_MIN_STACK 32*1024
-#define PCRE_JIT_MAX_STACK 512*1024
-
-#endif
+//TODOpcre2 pcre2_jit_stack_create ?
```

Comment:
Is this relevant ?
Is this a question of performance ? If so, how do I benchmark it ?